walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html

```diff
@@ -1,13 +1,40 @@
 <!DOCTYPE html>
```

Comment:
You can take a look at recommended styling guidelines for Google HTML projects at https://google.github.io/styleguide/htmlcssguide.html. Some noteworthy items:
- the indentation is recommended to be 2 spaces (I see 4 spaces in some places)
- Breaking long lines is optional for HTML, but this is a good practice to follow as it applies to most languages / projects. Github requires to scroll to the right to see lines that are longer than 125 characters, so it's a good practice to break the line before then.